akka-cluster-client-grpc-scala/.gitignore

```diff
@@ -0,0 +1,20 @@
+*#
```

Comment:
nitpick: I'd say it's best practice to only put things in here that are generated by the project and leave all OS/environment specific things to `.git/info/excludes` or your global git configs.

Reply:
That is from https://github.com/patriknw/akka-cluster-client-grpc/blob/wip-spike2-patriknw/.gitignore#L1 but I'll rm.